### ISSUE TITLE
This is how all the stacks work together

### ISSUE THREAD

**Issue by rix0rrr** [NetaNir/meerkats#15]
![image](https://user-images.githubusercontent.com/524162/75890220-56716500-5e2e-11ea-938a-9fc8174e6c58.png)

**Comment by rix0rrr** [comment]
We still need all of them even if we get rid of **Beta** and **Gamma**: 

![image](https://user-images.githubusercontent.com/524162/75890398-99333d00-5e2e-11ea-9ea2-85df4c25aaf7.png)

**Comment by rix0rrr** [comment]
Although we might be able to hack it if we use the Deployment Role cross-region:

![image](https://user-images.githubusercontent.com/524162/75890815-29718200-5e2f-11ea-92c2-e377ec7f6130.png)

**Comment by skinny85** [comment]
> Although we might be able to hack it if we use the Deployment Role cross-region:

Yes. I think we should try to avoid that second bootstrap stack in the Pipeline account (in `eu-west-1`).
